doc/conf.py

```diff
@@ -45,9 +45,26 @@
     "sphinx.ext.intersphinx",
     "sphinx_automodapi.automodapi",
     "sphinx_copybutton",
+    "sphinxext.opengraph",
     "m2r2",
 ]
 
+# Open Graph metadata
+ogp_title = "PennyLane Documentation"
+ogp_type = "website"
+ogp_description = "PennyLane is a cross-platform Python library for differentiable programming of quantum computers. Train a quantum computer the same way as a neural network."
+ogp_social_cards = {
+    "image": "_static/logo.png",
+    "enable": True,
+    "site_url": "https://docs.pennylane.ai/",
+    "line_color": "#03b2ff",
+}
+ogp_image = "_static/header-tall.png"
```

Comment:
There is some cutoff with the tall version:

![image](https://github.com/PennyLaneAI/pennylane/assets/49409390/60dc3621-a399-450f-a51e-a25a0ab48a91)

Did you try the not-tall version?

Reply:
Unfortunately neither seemed to work :( We need a version properly sized for OG, but I think that can always come later